docs/deployment/README.md#cumulus-instance-sizing

```diff
@@ -422,9 +422,20 @@ Notes on specific variables:
 
 Consider [the sizing of your Cumulus instance](#cumulus-instance-sizing) when configuring your variables.
 
-#### Configure the Thin Egress App
+#### Choose a distribution API
```

Comment:
Why is this file called README.md#cumulus-instance-sizing  🤔  ?

Reply:
Yup. No idea. I was thinking it was some weird docasaurus convention. I can't find a good reason why though and naming it README.md doesn't seem to break anything so...

36a798437d75720e24b4fb6347ff62a97626afd5